src/main/java/entities/DrugList.java

```diff
@@ -0,0 +1,18 @@
+package entities;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * A list containing information about all available drugs.
+ */
+public class DrugList {
+    /**
+     * @return A list containing information about all available drugs.
+     */
+    public static List<DrugInfo> getDrugList() {
+        return drugList;
+    }
```

Comment:
Do we really need a list of all the drugs info at a single time haha?

Reply:
It's not necessary, just might be nice to have; I'm open to changing it.